environments/staging/staging-hearings-reform-hmcts-net.yml

```diff
@@ -0,0 +1,16 @@
+name: "staging-hearings-reform.hmcts.net"
```

Comment:
```suggestion
name: "staging.hearings.reform.hmcts.net"
```

Reply:
Corrected for staging to remove "-". Staging commented out first to allow prod to deploy hearings.reform.hmcts.net first.